### PR TITLE
Show Slur or Tie instead of Slur/Tie in inspector.

### PR DESCRIPTION
**Pull request by sraduvictor**
This fixes one issue from https://musescore.org/en/node/100996

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
On top you could make the "Slur/Tie" test in inspector_slur.ui file untranslatable, as nobody would ever get to see it anyway? Unless:maybe if slurs and ties are selected? What happens in that case?

**Comment by Jojo-Schmitz** [comment]
On top you could make the "Slur/Tie" test in inspector_slur.ui file untranslatable, as nobody would ever get to see it anyway? Unless maybe if slurs and ties are selected at the same time? What happens in that case?
Also it is better to include "fix #100996" in the commit message, this causes the issue to automagically get marked fixe when the PR gets merged

**Comment by sraduvictor** [comment]
@Jojo-Schmitz , I'll change the code to have Slur/Tie appear when selecting both a slur and a tie.

**Comment by Jojo-Schmitz** [comment]
Just don't override the text from the ui file in that case might be easiest

**Comment by sraduvictor** [comment]
I tried leaving the original text for mixed selection, but that didn't work.

File: mscore/inspector/inspector_slur.ui [hunk 1]
```diff
@@ -41,7 +41,7 @@
       </font>
      </property>
      <property name="text">
-      <string>Slur/Tie</string>
+      <string></string>
```
Comment (Jojo-Schmitz) [inline mscore/inspector/inspector_slur.ui]: You can leave that string in, so GUI designers still see to, but mark it not to get translated, so translataors don't get bothered with it.
`<string notr="true">Slur/Tie</string>`

**Comment by Jojo-Schmitz** [comment]
Travis failes, with some exception in an mtest.

**Comment by sraduvictor** [comment]
I ran the mtest locally and it passed. I changed a conversion from (int) to static_cast<int> and reuploaded the diff.

**Comment by Jojo-Schmitz** [comment]
The travis failure most likely is not your fault at all

**Pull request merged**